docs/src/components/AppRouter.js

```diff
@@ -126,15 +126,29 @@ export default function AppRouter() {
           />
         </Route>
         <Route
-          title="Layout"
-          path="/layout"
+          title="Responsive UI"
+          path="/responsive"
```

Comment:
Their segments are `Layout/Responsive UI/Grid`.

We could change the top to `Layout` to match theirs, but I think we need to rename our `Layout` to `Grid`, which also more closely matches their terminology.  Then my `basics.md` can change to `responsive-ui.md` with `grid.md` and `hidden.md` as siblings.

Thoughts?